nilearn/plotting/displays.py

```diff
@@ -218,6 +236,89 @@ def draw_position(self, size, bg_color, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def _add_graph_nodes(self, nodes_coords, node_color, node_size, **kwargs):
+        """Plot graph nodes"""
+        if 's' in kwargs:
+            raise ValueError("Please use 'node_size' and not 'node_kwargs' "
+                             "to specify node sizes")
+        if 'c' in kwargs:
+            raise ValueError("Please use 'node_color' and not 'node_kwargs' "
+                             "to specify node colors")
+        if node_size is None:
+            node_size = 50
+        if node_color is None:
+            node_color = 'green'
+
+        nodes_coords_2d = _coords_3d_to_2d(nodes_coords, self.direction)
+
+        first_coords, second_coords = zip(*nodes_coords_2d)
+
+        defaults = {'marker': 'o',
+                    'zorder': 1000}
+        for k, v in defaults.items():
+            kwargs.setdefault(k, v)
+
+        self.ax.scatter(first_coords, second_coords, s=node_size, c=node_color,
```

Comment:
Somewhat general comment: How about adding an option for a depth filter. E.g. when looking at the sagittal / medial glass brain, why not make it possible to not show the other / not directly looked at half of the brain? This should be easy to implement (re-filter the coordinate pools before scatter) and might be especially useful if many nodes are to be visualized.